### PR TITLE
Show all the bootstrap candidates on the results, even if they couldn't

### PR DESCRIPTION
**Pull request by whirm**
be contacted.

### CODE REVIEW

**Comment by whirm** [comment]
Why are the vu trackers still getting there? are the trackers caching it somehow?
Also, why is prox1 showing as catalan???

```
cat summary.txt 
HOST_NAME ADDRESS REQUESTS RESPONSES
superpeer9.das2.ewi.tudelft.nl 130.161.211.194:6427 10 10
superpeer9.das2.ewi.tudelft.nl 130.161.211.194:6428 10 10
asmat.das2.ewi.tudelft.nl 130.161.211.199:6425 10 10
asmat.das2.ewi.tudelft.nl 130.161.211.199:6426 10 10
kayapo.das2.ewi.tudelft.nl 130.161.211.245:6421 10 0
kayapo.das2.ewi.tudelft.nl 130.161.211.245:6422 10 10
om.cs.vu.nl 130.37.198.19:6423 10 0
om.cs.vu.nl 130.37.198.19:6424 10 0
catalan.das2.ewi.tudelft.nl 95.211.198.141:6423 10 0
prox2.tribler.org 95.211.198.143:6424 10 10
```

```
host 95.211.198.141
141.198.211.95.in-addr.arpa domain name pointer prox1.tribler.org
```

**Comment by NielsZeilemaker** [comment]
@whirm trackers aren't cached, have a look at bootstrap.py to see which trackers are dns resolved.
How the host is reverse dns-ed, I don't know.

File: tests/test_bootstrap.py [hunk 1]
```diff
@@ -124,21 +125,8 @@ def __init__(self, *args, **kargs):
...
-                super(PingCommunity, self)._initialize_meta_messages()
-
             @inlineCallbacks
             def start_walking(self):
-                ping_count = 10
-                assert_margin = 0.9
-                max_rtt = 0.5
-
                 for _ in xrange(10):
                     self._pcandidates = list(self._iter_categories((u"discovered",), once=True))
```
Comment (NielsZeilemaker) [inline tests/test_bootstrap.py]:
```
if self._dispery._discovery_community and self._dispersy._discovery_community.bootstrap.are_resolved:
    self._pcandidates = self._dispersy._discovery_community.bootstrap.candidate_addresses
```

**Comment by whirm** [comment]
@NielsZeilemaker i'd say it's ready to merge

**Pull request merged**